kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/metrics/TransactionReportSuite.scala

```diff
@@ -267,6 +267,8 @@ class TransactionReportSuite extends AnyFunSuite with MetricsReportTestUtils {
   }
 
   test("TransactionReport: manually commit a remove file") {
+    // TODO: re-enable this test once remove file is supported.
```

Comment:
When we add fileSizeHistogram to transactionMetrics let's be sure to update these tests in this suite